health/micro-ui/web/docker/Dockerfile

```
@@ -1,19 +1,27 @@
-RUN cd web/ \
-    && chmod +x install-deps.sh \
-    && yarn install \
-    && yarn build:webpack 
+# Set working directory
+WORKDIR /app/web
+
+RUN chmod +x ./install-deps.sh
+RUN ./install-deps.sh
+
+# Increase timeout before install (recommended for GitHub Actions)
+RUN yarn config set network-timeout 600000
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)